Multicluster tutorial (#681)
kiali/kiali.io: content/en/docs/Tutorials/multicluster/06-Install-Travels-east-cluster.md

weight: 6
---

Run the following commands to install Travels application on east cluster:

```
kubectl create namespace travel-agency --context $CLUSTER_EAST
kubectl create namespace travel-portal --context $CLUSTER_EAST
kubectl create namespace travel-control --context $CLUSTER_EAST

kubectl label namespace travel-agency istio-injection=enabled --context $CLUSTER_EAST
kubectl label namespace travel-portal istio-injection=enabled --context $CLUSTER_EAST
kubectl label namespace travel-control istio-injection=enabled --context $CLUSTER_EAST

kubectl apply -f <(curl -L https://raw.githubusercontent.com/kiali/demos/master/travels/travel_agency.yaml) -n travel-agency --context $CLUSTER_EAST
kubectl apply -f <(curl -L https://raw.githubusercontent.com/kiali/demos/master/travels/travel_portal.yaml) -n travel-portal --context $CLUSTER_EAST
kubectl apply -f <(curl -L https://raw.githubusercontent.com/kiali/demos/master/travels/travel_control.yaml) -n travel-control --context $CLUSTER_EAST
```

After the installation, we can see that the Travels application is running on the east cluster:

![Overview](/images/mc-tutorial/01.png "Overview")

> It is important to note that Kiali only observes one istio-system namespace as we don’t configure it for multicluster yet.

Reviewer: ```suggestion
It is important to note that Kiali only observes one istio-system namespace as we did not configure it for multicluster yet.
```

> Take a look at the graph too to observe the in cluster traffic:

Reviewer: You should tell the reader that they need to select the three travel namespaces, because by default going to the  Graph page won't show them anything and they will be confused.

```suggestion
Go to the Graph page and select the three namespaces related to the Travels demo in the namespace dropdown menu. This shows you the in-cluster traffic:
```

![Graph](/images/mc-tutorial/02.png "Graph")

So far, we installed everything on one cluster, similarly to the Travels tutorial for a single cluster.

> Now we will expand this topology to include a remote cluster. As we commented this situation can be very common in a productive scenario, either because we might want to split some applications into different clusters, generally because they are maintained by different developers or for high availability or just making applications available in other zones to reduce latencies.

Reviewer: ```suggestion
Now we will expand this topology to include a remote cluster. As we commented this situation can be very common in a production scenario, either because we might want to split some applications into different clusters, generally because they are maintained by different developers or for high availability or just making applications available in other zones to reduce latencies.
```

> Now we will expand this topology to include a remote cluster. As we commented this situation can be very common in a production scenario, either because we might want to split some applications into different clusters, generally because they are maintained by different developers or for high availability or just making applications available in other zones to reduce latencies.

Reviewer 2: As with the intro, I'm not sure we need to include motivations for adopting primary-remote here in the tutorial.

Author: I think it's ok to give people some sort of use cases, but I'm ok if you prefer to remove it. I'm working on a blog post that might include thise motivations.
